.gitignore

```diff
@@ -11,3 +11,4 @@ output/
 tmp/
 build/
 config/
+.idea/
```

Comment:
Please remove this bit as it's editor specific. You might want to :eye: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.

Reply:
Thanks, I did not know that. Reverted.